ember-cli-build.js

```diff
@@ -69,7 +68,7 @@ module.exports = function(defaults) {
         },
         sassOptions: {
             includePaths: [
-                'node_modules/@centerforopenscience/osf-style/sass',
+                'app/styles/',
```

Comment:
This was the only way I could find to get addons to be able to import the variables file. I suspect the previous way of doing it was causing us to import the osf-style variables rather than our own, but I'm not 100% on that.